acmed/src/duration.rs

```diff
@@ -2,6 +2,7 @@ use acme_common::error::Error;
 use nom::bytes::complete::take_while_m_n;
```

Comment:
Can you explain this change?

Reply:
This fixes a defect found. The fuzzer found a panic due to integer overflow so I changed it to the checked version so it could run efficiently and explore more of the program